File: solutions/TypeScript/26.RemoveDuplicatesFromSortedArray/RemoveDuplicatesFromSortedArray.ts
function removeDuplicates(nums: number[]): number {
  if (nums.length === 0) return 0;

  let index: number = 1;

  for (let i: number = 1; i < nums.length; i++) {
    if (nums[i] != nums[i - 1]) {
      nums[index] = nums[i];
      index++;
    }
  }

  return index;
}

File: solutions/Python/26.RemoveDuplicatesFromSortedArray/RemoveDuplicatesFromSortedArray.py
# Imports not needed in Leetcode
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0

        index = 1

        for i in range(1, len(nums)):
            if nums[i] != nums[i - 1]:
                nums[index] = nums[i]
                index += 1

        return index
